azavea/iow-boundary-tool
# Wire up Boundary List Page

## Overview

Wires up the boundary list page to the API.

This fetches the boundaries for the current user and shows them in the table. Each row is clickable to see submission details. The data is sorted most recent first. Appropriate details are filled in to the table. There is a spinner shown while fetching. An error message is displayed in case of failure.

Builds on top of #113, will repoint to `develop` once that is merged.

Closes #87 

### Demo

<img width="1109" alt="image" src="https://user-images.githubusercontent.com/1430060/195484060-72b3dd3f-845a-43ab-9456-68e1a249b99f.png">

### Notes

We may need to rethink the IN REVIEW status. It may be extraneous.

Currently we fetch all the boundaries for the contributor user. Filter by utility will come in an future PR.

## Testing Instructions

- Check out this branch and `update` and `server`
- Go to http://localhost:4545/submissions and log in as a contributor
  - [ ] Ensure you see a list of test boundaries, with appropriate dates and statuses
  - [ ] Ensure clicking a row takes you to the appropriate detail page (yet to be wired up)

## Checklist
